deploy/chart/values.yaml

```
@@ -46,10 +53,13 @@ deploy:
   clusterID: null  # will be injected on installation on ACK
   regionID: null  # useful when deploying on non-ECS, but need to access Alibaba Cloud OpenAPI
 
-  # To access Alibaba Cloud OpenAPI in self-deployed Kubernetes cluster, use a secret with "id" and "secret" set.
+  # To access Alibaba Cloud OpenAPI in self-deployed Kubernetes cluster, use a Secret with "id" and "secret" set.
+  # Customize the keys in the Secret by setting the idField and secretField.
   accessKey:
     enabled: false
     secretName: csi-access-key
+    idField: id
+    secretField: secret
```

```suggestion
    idKey: id
    secretKey: secret
```
as it is used in `secretKeyRef.key`